OFEP-metric-hooks.md

```diff
@@ -0,0 +1,67 @@
+## OFEP-single-context-paradigm
```

Comment:
This looks like a copy/paste issue

Reply:
Thank you, yes, this was while extracting the format of the ofep.

Reply:
Resolved with the latest commit